charms/docker/compose.py

```diff
@@ -47,6 +47,19 @@ def kill(self, service=None):
             cmd = "docker-compose kill"
         self.run(cmd)
 
+    def rm(self, service=None):
+        '''
+        Convenience method that wraps `docker-compose rm`
+
+        usage: c.rm('nginx') to remove the 'nginx' service from the
```

Comment:
I disagree because this is the default behavior of docker-compose. This is a convention that is super-imposed by the docker tool chains, and if we deviate from it, the syntax of our wrapper class does not accurately reflect what the tooling is doing.

As an added bonus, you have to c.kill() before you can c.rm() any of the running app containers. so either you have crap code in 2 places or you intended to kill and remove everything :)